jordyob03/TripTailor
# Tt 78 at image saving

# Added image saving

users can upload images
the images are associated with events which are associated with itins, they are connected through ids

## To Test 
create itins, check pgadmin make sure data for itin events and images are there and are linked, check if images are hosted, if not know how to do, refer to 
https://github.com/jordyob03/TripTailor/pull/27
